Bonsai.Harp.CF/SyringePumpCommand.cs

```diff
@@ -7,11 +7,11 @@ namespace Bonsai.Harp.CF
 {
     public enum SyringePumpCommandType : byte
     {
-        EnableMotorDriver,
-        DisableMotorDriver,
+        SetMotorDriver,
 
-        StartProtocol,
-        StopProtocol,
+        SetDirection,
+
+        EnableProtocol,
```

Comment:
This is okay, but the below is general feedback to take into account.

In the past we used Enable/Disable and Start/Stop commands explicitly to allow creating the message from any input type (instead of requiring a dependency on a specific input value). To keep consistency with the other devices in this repository this pattern would be kept.

We do have an alternative solution coming up with Bonsai 2.7 (coming out in the fall) where it becomes possible to parameterize input values for each command type as properties of the node (essentially allowing for polymorphic commands), but this is going to be take at least one more month to be released probably.